### ISSUE TITLE
Add progress indicator during database generation

### ISSUE THREAD

**Issue by romainsacchi** [polca/ScenarioLink#2]

**Comment by marc-vdm** [comment]
Could you point me to the functions in unfold where the progressbars are printed to the terminal?

I've looked at the AB progressbars, it seems we overwrite the classes in Brightway that take the different steps (extract/apply strategies etc) and update progress bar from there. Perhaps we could do something similar to this for unfold if it's not too complicated.

**Comment by romainsacchi** [comment]
It happens here: https://github.com/polca/ScenarioLink/blob/04fecb2f820012507d9ec86104b32db76d50aab0/ab_plugin_scenariolink/utils.py#L41

**Comment by marc-vdm** [comment]
Yes, sorry, I mean the specific functions in [unfold](https://github.com/polca/unfold/blob/a35f6c799e25e1c862f5921f503c232664480828/unfold/unfold.py#L94)

**Comment by romainsacchi** [comment]
That'd be this one: https://github.com/polca/unfold/blob/a35f6c799e25e1c862f5921f503c232664480828/unfold/unfold.py#L1116

**Comment by bsteubing** [comment]
would be nice to have... the download now took some 4-5 minutes, so a progress bar would be great. And it is unclear where the data is stored. Perhaps the user could directly indicate where he wants to store it?
